shared/src/business/entities/trialSessions/TrialSession.js

```diff
@@ -481,11 +511,12 @@ const isStandaloneRemoteSession = function (sessionScope) {
  * @returns {TrialSession} the trial session entity
  */
 TrialSession.prototype.setAsClosed = function () {
-  this.isClosed = true;
```

Comment:
It looks like there are other places in the code where setting `isClosed` has been removed. I'm good with that and think it's a great idea but in that case, I think that value should be removed off the entity too, since we can no longer rely on it being a valid value.

Reply:
Forgive me if I'm missing something, but it has been removed, up on what was line 56. It's also removed in the migration

Reply:
I just removed it from the DTO, but there should only be one place is exists at this point which is in the helper function used in the data migration